src/main.c

```diff
@@ -69,17 +69,51 @@ void connect_console_command(char *str) {
         g->typing = 1;
         g->typing_buffer[0] = '\0';
     } else if (g->server_user[0] == '\0') {
-        strncpy(g->server_user, str, 32);
-        snprintf(g->text_message, KONSTRUCTS_TEXT_MESSAGE_SIZE,
-                "Enter a password to login/create account");
-        snprintf(g->text_prompt, KONSTRUCTS_TEXT_MESSAGE_SIZE, "Password");
+        char new_str[strlen(str)+1]; // Strip all whitespaces away
```

Comment:
Er, one question: The model::server_pass array has a size of 64, but in main.c, there is only a maximum size of 32.
Now the question: does the server support 64-char passwords? If so, I'll change it ok?